test/integration/entry-integration.js

```diff
@@ -419,4 +419,131 @@ describe('Entry Api', () => {
       expect(response.items[0].sys.publishedAt).to.not.be.undefined
     })
   })
+
+  /**
+   * This test was created to make sure the SDK supports
+   * creating/updating/publishing/upublishing/deleting of
+   * entries with cross space links
+   *
+   * This test is slightly fragile as it can break if the
+   * entry '4zimQzVMxDsSX607PCfo2u' from the space '6mqcevu5a50r' is deleted
+   */
+  describe('write with xspace references', () => {
+    let xSpaceEnabledSpace
+    let xSpaceEnabledEnvironment
+    let xSpaceDisabledSpace
+    let xSpaceDisabledEnvironment
+    let contentTypeDisabledSpace
+    let contentTypeEnabledSpace
+    let contentTypeData = {
+      name: 'testCTXSpace',
+      fields: [
+        {
+          id: 'title',
+          name: 'Title',
+          type: 'Text',
+        },
+        {
+          id: 'multiRefXSpace',
+          name: 'multiRefXSpace',
+          type: 'Array',
+          items: {
+            type: 'ResourceLink',
+            validations: [],
+          },
+          allowedResources: [
+            {
+              type: 'Contentful:Entry',
+              source: 'crn:contentful:::content:spaces/6mqcevu5a50r',
+              contentTypes: ['testCtxSpace'],
+            },
+          ],
+        },
+      ],
+    }
+
+    let entryData = {
+      fields: {
+        title: { 'en-US': 'this is the title' },
+        multiRefXSpace: {
+          'en-US': [
+            {
+              sys: {
+                type: 'ResourceLink',
+                linkType: 'Contentful:Entry',
+                urn: 'crn:contentful:::content:spaces/6mqcevu5a50r/entries/4zimQzVMxDsSX607PCfo2u',
+              },
+            },
+          ],
+        },
+      },
+    }
+
+    before(async () => {
+      // default space has xspace enabled in it through the feature flags
+      xSpaceEnabledSpace = await getDefaultSpace()
+      xSpaceEnabledEnvironment = await xSpaceEnabledSpace.getEnvironment('master')
+      xSpaceDisabledSpace = await createTestSpace(initClient(), 'Entry')
+      xSpaceDisabledEnvironment = await createTestEnvironment(
+        xSpaceDisabledSpace,
+        'Testing Environment'
+      )
+      contentTypeDisabledSpace = await xSpaceDisabledEnvironment.createContentTypeWithId(
```

Comment:
This reads like a space for me but it is actually a content type. Maybe `xSpaceDisabledContentType`. Though this comment is subjective.